### PR TITLE
Fix for dict.viewitems(), dict.iteritems() etc in chained calls.

### LINKED ISSUE #169: python-modernize breaks dict.itervalues().next()
python-modernize breaks `dict.itervalues().next()`.

It changes the code to `six.itervalues(dict)` instead of `next(six.itervalues(dict))`.

### PR DESCRIPTION
**Pull request by martindemello**
Fixes #169

### CODE REVIEW

File: tests/test_fix_dict_six.py [hunk 1]
```diff
@@ -1,6 +1,6 @@
 from __future__ import absolute_import
 
-from utils import check_on_input
+from libmodernize.tests.utils import check_on_input
```
Comment (takluyver) [inline tests/test_fix_dict_six.py]: The tests aren't actually part of libmodernize, so this is causing a failure.

Comment (martindemello) [reply tests/test_fix_dict_six.py]: sorry, forgot to revert that before pushing!

**Comment by firba1** [comment]
It would be nice if this could be merged, I'm running into this issue.

File: libmodernize/fixes/fix_dict_six.py [hunk 2]
```diff
@@ -3,27 +3,48 @@
...
+        head = results['head']
+        method = results['method'][0] # Extract node for method name
+        tail = results['tail']
+        syms = self.syms
+        method_name = method.value
         name = fixer_util.Name(u'six.' + method_name, prefix=node.prefix)
-        node.replace(fixer_util.Call(name, new_node))
+        isiter = method_name.startswith(u'iter')
+        isview = method_name.startswith(u'view')
+        assert isiter or isview, repr(method)
```
[the remark this answers is not in the document]
Comment (martindemello) [reply libmodernize/fixes/fix_dict_six.py]: didn't know startswith took a tuple! done.

**Comment by takluyver** [comment]
Thanks!

**Pull request merged**